src/Sylius/Bundle/CoreBundle/Form/Type/ImageType.php

```diff
@@ -14,7 +14,7 @@
 use Sylius\Bundle\ResourceBundle\Form\Type\AbstractResourceType;
 use Symfony\Component\Form\FormBuilderInterface;
 
-class ImageType extends AbstractResourceType
+abstract class ImageType extends AbstractResourceType
```

Comment:
Removed that abstract, registering this one as `sylius_product_variant_image`. `sylius_image` has no meaning right now as we don't have a general model for images (just `ProductVariantImage`, `TaxonImage`, etc.).